drivers/i2s/i2s_mcux_sai.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021,2023-2024 NXP Semiconductor INC.
```

Comment:
Commit subject should say:  `drivers:audio: i2s: Enable SAI mclk`

`Update ... ` is not a real information.